packages/ember-htmlbars/lib/main.js

```diff
@@ -147,6 +148,7 @@ registerHelper('-normalize-class', normalizeClassHelper);
 registerHelper('concat', concatHelper);
 registerHelper('-join-classes', joinClassesHelper);
 registerHelper('-html-safe', htmlSafeHelper);
+registerHelper('hash', hashHelper);
```

Comment:
Should be feature flagged (just the one flag for contextual components is good enough IMO)